frontend/src/components/widgets/FileUploader/FileUploader.tsx

```diff
@@ -145,6 +145,11 @@ class FileUploader extends React.PureComponent<Props, State> {
   public componentDidUpdate = (prevProps: Props): void => {
     const { element, widgetMgr } = this.props
 
+    // TODO(vdonato): Rework this now that there's a short window where the app
+    // may reconnect to the server without losing its uploaded files. Just
+    // removing this seemed to work, but I'm not entirely sure if it's the
+    // correct fix.
+    //
```

Comment:
We have a similar code for camera input too https://github.com/streamlit/streamlit/blob/a5e814127d0387cb5959d64739a1b0a0ca0e2df5/frontend/src/components/widgets/CameraInput/CameraInput.tsx#L264

Reply:
Ah, yeah -- let me add a similar comment there so we don't forget to also fix this behavior for camera input